src/binder/bind/bind_ddl.cpp

```diff
@@ -174,9 +177,21 @@ std::unique_ptr<BoundStatement> Binder::bindCreateTable(const Statement& stateme
     return std::make_unique<BoundCreateTable>(std::move(boundCreateInfo));
 }
 
+std::unique_ptr<BoundStatement> Binder::bindCreateSequence(const Statement& statement) {
+    auto& createSequence = ku_dynamic_cast<const Statement&, const CreateSequence&>(statement);
+    auto info = createSequence.getInfo();
+    auto sequenceName = info->sequenceName;
+    if (clientContext->getCatalog()->containsSequence(clientContext->getTx(), sequenceName)) {
+        throw BinderException(sequenceName + " already exists in catalog.");
+    }
+    auto boundInfo = BoundCreateSequenceInfo(sequenceName, info->startWith, info->increment,
```

Comment:
I think you should also do some basic checks over the definition of the sequence, such as that `minValue` should be smaller or equal to `maxValue`, and `startWith` should be within the range of `minValue` and `maxValue`. Also, add test cases for those exceptions.